tests/e2e/models/Navigation.js

```diff
@@ -0,0 +1,32 @@
+import { Selector } from 'testcafe';
+
+/**
+ * Navigation Model
+ * @class
+ * @classdesc defines Navigation base elements
+ */
+export default class Navigation {
```

Comment:
No, I would like room for the Navigation to be its own thing - because it is.  And it changes shape depending on screen width.  It will help tremendously as we move to test mobile navigation.